File: backend/src/main/java/ch/egli/streef/trackpoint/TrackPointDto.java
package ch.egli.streef.trackpoint;

import java.time.LocalDateTime;

import lombok.Data;

@Data
public class TrackPointDto {
    private Long id;
    private Long trackId;
    private double latitude;
    private double longitude;
    private double height;
    private LocalDateTime timestamp;
    private LocalDateTime createdAt;
}

File: frontend/src/app/track/track-point.dto.ts
export interface TrackPointDto {
    id: number;
    trackId: number;
    latitude: number;
    longitude: number;
    height?: number | null;
    timestamp: string; // ISO string (e.g., "2025-05-13T12:34:56")
    createdAt: string;
}
